src/kernel/retype_tbl.c

```diff
@@ -238,15 +480,18 @@ retype_tbl_init(void)
 	assert(CACHE_LINE % sizeof(struct retype_entry) == 0);
 
 	for (i = 0; i < NUM_CPU; i++) {
-		for (j = 0; j < N_MEM_SETS; j++) {
+		for (j = 0; j < (N_MEM_SETS << 1); j++) {
```

Comment:
I've read through a lot of this, and I still don't understand this.

Reply:
I will give you a email tonight explaining this in detail with figs.